prometheus-ksonnet/jsonnetfile.json

```diff
@@ -8,7 +8,7 @@
                     "subdir": ""
                 }
             },
-            "version": "release-0.1"
+            "version": "master"
```

Comment:
nit: this sounds like a floating label that might change quite a lot -- is that ok?

Reply:
The point is not a nit. As @malcolmholmes also pointed out to me, we should not force a new version just because we upgraded. Fortunately, this is not much of a problem because of how `jb` (0.2.0 at least) works: it doesn't by default install/vendor dependencies of dependencies, because of this it is likely a conscious decision of downstream to vendor it in or not.

Reply:
As to justify the changes specific to our environment, the overrides and the version 'release-0.1' here were originally because we didn't upgrade. This PR should set things right.

Reply:
Yep, this was a locking back for Grafana Labs. This sets this right.